client/src/containers/CompareContainer.js

```diff
@@ -27,6 +30,26 @@ function CompareContainer(props) {
     return catSettings.includes(o.espnId) && o.name !== 'mins'
   })
 
+  // Calculating Head to Head matchup table
+  if (!selectedTeams.includes('')) {
+    const h2h_row_1 = {}
+    const h2h_row_2 = {}
+    const teamOneId = parseInt(selectedTeams[0])
+    const teamTwoId = parseInt(selectedTeams[1])
+    h2h_row_1['rowHeader']= teams.filter((team) => team.teamId === teamOneId)?.[0]?.fullTeamName;
```

Comment:
If you use vscode, I use the prettier extension for all js formatting, it would fix the spacing around the =